app/services/support_interface/candidate_email_send_counts_export.rb

```diff
@@ -1,18 +1,33 @@
 module SupportInterface
   class CandidateEmailSendCountsExport
     def data_for_export
-      emails_with_counts.map do |email_hash|
+      emails_with_total_send_counts.map do |send_count_record|
+        unique_recipients = unique_recipients_hash[send_count_record.mail_template]
         {
-          'Email' => email_hash['mail_template'],
-          'Send count' => email_hash['send_count'],
-          'Last sent at' => email_hash['last_sent_at'],
+          'Email' => send_count_record.mail_template,
+          'Send count' => send_count_record.send_count,
+          'Last sent at' => send_count_record.last_sent_at,
+          'Unique recipients' => unique_recipients,
         }
       end
     end
 
   private
 
-    def emails_with_counts
+    def unique_recipients_per_email
+      distinct_mail_templates_and_recipients = Email.select('DISTINCT "to", "mail_template"').to_sql
+      Email
+        .select('mail_template, COUNT(mail_template) AS unique_recipients')
+        .from("(#{distinct_mail_templates_and_recipients}) AS template_recipient_combos GROUP BY template_recipient_combos.mail_template")
```

Comment:
I think `Email.select('count(distinct "to"), mail_template').group(:mail_template)` works too. Wondering whether you could just add the `count(distinct "to")` in the main query and drop this one altogether.

Reply:
I think you're absolutely right, thanks for the suggestion! New commit added.